### PR TITLE
Avoid marking configuration files as executable

### PR DESCRIPTION
**Pull request by sysrich**
**What this PR does / why we need it**:
The current reference rpm spec files install every file with their executable bit set, even when the files are not executable. 

This PR tidies this up so configuration files are not marked as executable when installed via these rpms.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by neolit123** [comment]
LGTM i verified this briefly last week.
thanks @sysrich

**Comment by sysrich** [comment]
Fixed the merge conflict

**Comment by timothysc** [comment]
/assign @ixdy @cblecker @spiffxp 

requires approver.

File: build/rpms/kubeadm.spec [hunk 1]
```diff
@@ -15,12 +15,12 @@ Command-line utility for deploying a Kubernetes cluster.
 
 %install
 install -m 755 -d %{buildroot}%{_bindir}
-install -m 755 -d %{buildroot}%{_sysconfdir}/systemd/system/
-install -m 755 -d %{buildroot}%{_sysconfdir}/systemd/system/kubelet.service.d/
-install -m 755 -d %{buildroot}%{_sysconfdir}/sysconfig/
+install -m 644 -d %{buildroot}%{_sysconfdir}/systemd/system/
```
Comment (ixdy) [inline build/rpms/kubeadm.spec]: directories should be 755, right?

**Review by ixdy — COMMENTED** [review]
/hold

File: build/rpms/kubeadm.spec (hunk 1 above)
Comment (sysrich) [reply build/rpms/kubeadm.spec]: Indeed, good point :blush: 

fixed! thanks @ixdy

**Comment by ixdy** [comment]
lgtm, but can you please rebase to remove the merge commit?

**Pull request merged**